src/app/(proper_react)/redesign/(authenticated)/user/welcome/FindExposures.tsx

```diff
@@ -67,7 +84,21 @@ export const FindExposures = ({
     progressRange: [labelSwitchThreshold, 100],
   });
 
+  const { gtag } = useGa();
+  const pathName = usePathname();
+
   useEffect(() => {
+    // view page telemetry
+    const findExposuresTelemetryParams: FindExposuresTelemetryParams = {
+      exit_time: 0,
+      page_location: pathName,
+      user_type: "non_legacy",
```

Comment:
Don't we need to detect whether the user's account is older than Plus is? AFAIK you can also get to this flow from the Guided Experience.

Reply:
Ah, thanks for the note. I assumed it's only from dashboard. It makes sense. I think we can detect this via `referrer` in the uri